### DISCUSSION TITLE
What does "Download Game Images" do?

### DISCUSSION THREAD

**Discussion by camerongoble** [tkashkin/GameHub#646] [General]
In the Add Games ("+") button, there is an option to "download game images".  A few questions:   

- What exactly does this do?  
- How can I tell where the downloads appear?  
- Is it grabbing installs of every game in my library, or what?  
- How do I stop it once it starts?

**Comment by tkashkin** [comment]
It downloads the grid images for games that don't have them from Steam, [SteamGridDB](https://steamgriddb.com), [steam.cryotank.net](https://steam.cryotank.net).
